wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -237,20 +237,7 @@ instance ByteArray.ByteArrayAccess a => ByteArray.ByteArrayAccess (V1 a) where
 
 -- TODO(adinapoli) Rewrite it properly under CSL-2048.
```

Comment:
@akegalj One for you: once we close the relevant ticket, we will need to remove this stale comment 😉